src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -3,7 +3,13 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, turf, _) {
 
     var defaults = {
         id: 'map',
-        selector: '#map',
+        selectors: {
+            id: '#map',
+            leafletMinimizer: '.leaflet-minimize',
+            leafletLayerList: '.leaflet-control-layers-list',
+            leafletLayerControl: '.leaflet-control-layers',
+            layerListMinimizedClass: 'minimized'
```

Comment:
Since there was no layer picker in the prototype, these classes don't exist in the redesigned styles.  Grabbing them from [the old version](https://github.com/azavea/cac-tripplanner/blob/develop/src/app/styles/templates/_app.scss#L25) makes this look a lot better.

![layer_picker](https://cloud.githubusercontent.com/assets/6598836/20348890/d6f82e3e-abd5-11e6-85e9-db424c76199b.png) ![layer_picker_minimized](https://cloud.githubusercontent.com/assets/6598836/20348895/d93a4358-abd5-11e6-9be5-5c1da359fa11.png)

Specifically that's with
```
    .leaflet-control-layers {
        margin-right: 30px;
        margin-bottom: 30px;
        position: relative;
        padding: 15px 20px 18px;
        border-radius: 0;
        box-shadow: 0 0 12px rgba(0,0,0,0.35);

        .leaflet-control-layers-separator {
            margin: 15px -20px 15px -20px;
        }
        .leaflet-control-layers-selector {
            margin-right: 8px;
            top: 0;
        }
    }

    .leaflet-minimize {
        color: #707070;
        cursor: pointer;
        position: absolute;
        border-radius: 3px;
        line-height: 10px;
        padding: 13px 15px;
        bottom: 0;
        z-index: 99999;
        right: 0;

        &.minimized {
            height: 38px;
            line-height: 0;
            background: white;
        }
    }
```
added to `src/app/styles/components/_map.scss`, which is almost the same as the old styles but with the font override removed and the height adjusted in `.minimized`.